src/irmin-pack/dict_intf.ml

```diff
@@ -33,5 +33,5 @@ end
 module type Dict = sig
   module type S = S
 
-  module Make (IO : IO.S) : S
+  module Make (_ : IO.VERSION) (IO : IO.S) : S
```

Comment:
I didn't know this trick, nice :) but I think it is a bit confusing, as we can also ignore the IO in this signature.

Reply:
In that case I will ignore IO too :slightly_smiling_face: We only recently got access to this, since it's a 4.08 feature.